### PR TITLE
GH-18 - Improve hashtag help text

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary
Specified reference date that must be included in the date format for the meeting hashtag. Follow up from PR https://github.com/mattermost/mattermost-plugin-agenda/pull/2

#### Screenshot

Screen Shot 2020-03-16 at 9 25 46 AM

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-agenda/issues/18

### CODE REVIEW

**Comment by marianunez** [comment]
Specific wording is open for suggestions

File: webapp/src/components/meeting_settings/meeting_settings.jsx [hunk 1]
```diff
@@ -117,12 +117,13 @@ export default class MeetingSettingsModal extends React.PureComponent {
                             className='form-control'
                             value={this.state.hashtag ? this.state.hashtag : ''}
                         />
-                        <p className='text-muted pt-1'> {'Date format should follow the one of  '}
+                        <p className='text-muted pt-1'> {'Date format must follow  '}
```
Comment (lieut-data) [inline webapp/src/components/meeting_settings/meeting_settings.jsx]: Proposed text, linking to Go Doc instead (as an authority?)

> Hash tag is formatted using the [Go time package](https://godoc.org/time#pkg-constants). Embed a date by writing down what January 2, 2006 would look like.

**Comment by marianunez** [comment]
Updated to: 

Screen Shot 2020-03-19 at 9 52 39 AM

**Comment by marianunez** [comment]
@lieut-data I mistakenly merged this PR to the review branch and the change is not present in the current version on community. 🤦‍♀️

I cherry-picked it and it's now available in `master` whenever a new version can be cut.

**Pull request merged**